Common/src/Storage/ITaskTable.cs

```diff
@@ -300,8 +312,25 @@ public interface ITaskTable : IInitializable
   /// <returns>
   ///   The expected output keys
   /// </returns>
-  Task<IEnumerable<string>> GetTaskExpectedOutputKeys(string            taskId,
-                                                      CancellationToken cancellationToken = default);
+  public async Task<IEnumerable<string>> GetTaskExpectedOutputKeys(string            taskId,
+                                                                   CancellationToken cancellationToken = default)
+  {
+    try
+    {
+      return await GetTasksExpectedOutputKeys(new[]
+                                              {
+                                                taskId,
+                                              },
+                                              cancellationToken)
+                   .Select(tuple => tuple.expectedOutputKeys)
+                   .SingleAsync(cancellationToken)
```

Comment:
`SingleAsync` throws if sequence is empty, but Excpection would not give the right information (ie: missing task ID).

Reply:
There is a catch for this exception !